weighted_levenshtein/__init__.py

```diff
@@ -1 +1 @@
-from clev import *
+from .clev import *
```

Comment:
nit: Add newline to the EOF

I realize that previously I asked you to remove the newline at the EOF, but it looks like you've removed two newlines. Each file should end with exactly one newline, no more, no less. So for example:
```
import os
```
is wrong
```
import os\n\n
```
is also wrong, but
```
import os\n
```
is correct. 👌 

I apologize for being so nit-picky, it's probably super annoying.

Reply:
no worries :)